### DISCUSSION TITLE
wiki/muiref web page displayed wrongly

### DISCUSSION THREAD

**Discussion by Claponi** [olikraus/u8g2#2511] [General]
Hi Olikraus,
just to inform you that there is a display issue on this web page https://github.com/olikraus/u8g2/wiki/muiref that make it difficult to read.
Problems start with "Numbers (uint8)" section.

Thank you in advance

![Screenshot 2024-09-18 171213](https://github.com/user-attachments/assets/befdaae7-0cef-4c8b-9444-1641a60631c6)

**Comment by olikraus** [comment]
For me this is very much readable: 
![grafik](https://github.com/user-attachments/assets/d987b16a-c847-45fa-8724-43ca61bb4f40)

**Comment by Claponi** [comment]
Thank you for your answer. 
Maybe I wasn't clear, the issue start after "numbers". 
For example : "bar graph", "fixed bar graph".... there is no animated Gif (most visible issue) and if you click on a link as "[List Selection (Child form, uint8, static list)](https://github.com/olikraus/u8g2/wiki/muiref#list-selection-child-form-uint8-static-list)" (inside the same problematic part) at the beginning of the page it's not working.

**Comment by olikraus** [comment]
Ah, got it. Fixed.
Looks like an issue with the github markup language :-/

Thanks for the info!
